ue5osc/__init__.py

```diff
@@ -41,9 +41,17 @@ def send_and_await(self, osc_address: str):
         return self.message_handler.wait_for_response()
 
     def get_project_name(self) -> str:
-        """Returns and optionally prints the name of the current connected project."""
+        """Returns the name of the current connected project."""
         return self.send_and_await("/get/project")
 
+    def get_raycast(self) -> str:
+        """Returns the length of the raycast."""
+        return self.send_and_await("/get/raycast")
+
+    def set_raycast(self, length: float) -> None:

```

Comment:
Should this be `set_raycast_length`? Should have this method, or should it always be based on how far forward we move in the `move_forward/backward` methods?

Reply:
It should be called set_raycast_length. Furthermore, I thought about just including it in the move_forward/backward methods, but wanted to return something so boxnav would be able to prevent the movement of the agent.